__tests__/NCN.test.js

```diff
@@ -421,5 +421,14 @@ describe('GET', () => {
                 }
```

Comment:
as there is no expect statement in the above test for if the body does equal 0 then the this test will pass by default

Reply:
i dont understand

Reply:
should i replace all my ifs for length checking with  this expect(body).not.toHaveLength(0)